src/Orleans/Messaging/IncomingMessageBuffer.cs

```diff
@@ -83,15 +91,10 @@ public bool TryDecodeMessage(out Message msg)
                 bodyLength = BitConverter.ToInt32(lengthBuffer, 4);
             }
 
-            // If message is too big for default buffer size, grow
+            // If message is too big for current buffer size, grow
             while (decodeOffset + KnownMessageSize() > currentBufferSize)
```

Comment:
few usages of the result for `KnownMessageSize()`. Consider storing in a local variable